active/operators/bort/run_ner_tasks.py

```diff
@@ -0,0 +1,42 @@
+import json
+from datetime import datetime
+
+import utilities.common_variables as common_variables
+import utilities.common_hooks as common_hooks
+import utilities.common_functions as common_functions
+#TODO: add prospective BERT endpoint in utilities?
+
+
+def run_bort_task(**kwargs):
```

Comment:
run_bort_task vs run_bort_tasks?
this function looks like it's going to run over multiple tasks (there's a loop)